index.html

```diff
@@ -163,5 +163,28 @@ <h3 class="skill-title">Skills</h3>
       </div>
     </section>
   </section>
+  <footer>
```

Comment:
[OPTIONAL] Kindly fix the position of the `background-image` to match the Figma design
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/160732910-8e381d4e-f069-459c-bd66-ae841c628b00.png) | ![image](https://user-images.githubusercontent.com/24703171/160733023-6c6c58fc-2ba7-4e23-b74f-4517b19fe528.png) |

`Hint:` give the background image to the footer instead of the section with class `contact-form` .

Reply:
Thank you!!